Lib/test/test_urlparse.py

```diff
@@ -417,6 +417,11 @@ def test_urljoins(self):
         self.checkJoin('svn+ssh://pathtorepo/dir1', 'dir2', 'svn+ssh://pathtorepo/dir2')
         self.checkJoin('ws://a/b','g','ws://a/g')
         self.checkJoin('wss://a/b','g','wss://a/g')
+        self.checkJoin(
+            'nonsensebase://net.loc/url/', '..',
+            'nonsensebase://net.loc/',
+            flags=(urllib.parse.SchemeFlag.RELATIVE | urllib.parse.SchemeFlag.NETLOC),
```

Comment:
Remove the `SchemeFlag` portion as its enums will be available at the global level.

For those that tend to forget, "global" means "module" in Python.